app/src/main/java/org/gdg/frisbee/android/chapter/NewsFragment.java

```diff
@@ -149,8 +150,9 @@ public void onGet(Object item) {
                     ActivityFeed feed = (ActivityFeed) item;
 
                     if (isAdded()) {
-                        Crouton.makeText(getActivity(), R.string.cached_content,
```

Comment:
Oh I see. It is because Crouton's uses `Activity`'s `View` by default. But here `Snackbar` can insert itself into `Fragment`'s `View`